Makefile

```diff
@@ -66,3 +68,11 @@ archive:
 	carthage archive SourceKittenFramework
 
 release: package archive
+
+spm_bootstrap:
+	cd /usr/local/include; svn export --force http://llvm.org/svn/llvm-project/cfe/trunk/include/clang-c/
+	cp clang_c_module.modulemap /usr/local/include/clang-c/module.modulemap
+	cp /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/libclang.dylib /usr/local/lib/
```

Comment:
swift-2.2-SNAPSHOT contains `libclang.dylib` at `/Library/Developer/Toolchains/swift-2.2-SNAPSHOT-2016-01-11-a.xctoolchain/usr/lib/libclang.dylib`. It would be better using that?